meta-balena-common/recipes-containers/balena-supervisor/balena-supervisor/update-balena-supervisor

```diff
@@ -27,6 +33,15 @@ while [ $# -gt 0 ]; do
             update_balena_supervisor_help
             exit 0
             ;;
+        -i|--supervisor-image)
+            if [ -z "$2" ]; then
```

Comment:
I think we should check the regex here or the following steps could fail in unexpected ways.

Something like this should match both?

```bash
# matches bh.cr/balena/supervisor/0.0.0
# matches registry2.balena-cloud.com/v2/8f3d658a373039a9c27b8f1811b0261d
REGEX='^[a-z0-9_\.-]+\/([a-z0-9_-]+\/[a-z0-9_-]+\/[a-z0-9_\.-]+|v2\/[a-z0-9_-]+)$'
```

Reply:
Sounds good to me!

Reply:
Documenting the final regex here as versionbot is particular with regex in git commits:
```
^((?!-)[A-Za-z0-9\-]{1,63}(?<!-)\.)+[A-Za-z]{2,6}\/[a-zA-Z0-9_\.\-\/]+|v[0-9]+\/[0-9a-zA-Z_\-]+$


^			                   # Start of line
 (			                   # Start of group
	(?! -)		               # Can't start with a hyphen
	[A-Za-z0-9-]{1,63}         # Domain name is [A-Za-z0-9-], between 1 and 63 chars long
	(?<!-)		               # Can't end with hyphen
	\\.		                   # Follow by a dot "."
 )+			                   # End of group, allowed more than once for subdomains
  [A-Za-z]{2,6}		           # Top Level Domain, between 2 and 6 chars long
 \/                            # Forward slash literal
 [a-zA-Z0-9_\.\-\/]+|v[0-9]+   # [a-zA-Z0-9_\.\-\/]+ Fleet name or v[0-9]+ versioned translation
 \/                            # Forward slash literal
 /[0-9a-zA-Z_\-]+              # semver or digest
$			                   #end of the line
```